dev-util/acr/acr-1.6.2.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="An autoconf replacement"
+DESCRIPTION="ACR tries to replace autoconf functionality generating a \
+full-compatible 'configure' script (runtime flags). But using shell-script \
+instead of m4. This means that ACR is faster, smaller and easy to use."
+HOMEPAGE="https://github.com/radare/acr"
+COPYRIGHT="2005-2018 pancake"
+LICENSE="GNU GPL v2"
+SOURCE_URI="https://github.com/radare/acr/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="12b2f5ceb4da4507fba5242fe5596b8e25916478709f1d6189cbc016b01e6661"
+REVISION="1"
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64 ?arm"
+
+PROVIDES="
+	acr = $portVersion
+	cmd:acr = $portVersion
+	cmd:acr_cat = $portVersion
+	cmd:acr_install = $portVersion
+	cmd:acr_sh = $portVersion
+	cmd:amr = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:find
+	cmd:make
+	cmd:sed
+	"
+
+BUILD()
```

Comment:
```diff
 	cmd:sed
 	"

+TEST_REQUIRES="
+	cmd:getopt
+	"
+
 BUILD()
```

Reply:
Thanks, wasn't sure about this one, first time use :)